contracts/airswap/AirSwapFeeConnector.sol

```diff
@@ -0,0 +1,202 @@
+pragma solidity 0.5.17;
+
+import "../openzeppelin/SafeMath.sol";
+import "../openzeppelin/PausableOz.sol";
+import "../openzeppelin/IERC20_.sol";
+
+import "./IAirSwapFeeConnector.sol";
+import "./ISwapERC20.sol";
+
+contract AirSwapFeeConnector is PausableOz, IAirSwapFeeConnector {
+    using SafeMath for uint256;
+
+    struct SwapRequest {
+        address sender;
+        address recipient;
+        uint256 nonce;
+        uint256 expiry;
+        address signerWallet;
+        address signerToken;
+        uint256 signerAmount;
+        address senderToken;
+        uint256 totalSenderAmount;
+        uint8 v;
+        bytes32 r;
+        bytes32 s;
+    }
+
+    uint256 public constant POINTS = 1000;
+
+    uint256 public inputFeeInPoints = 0;
+    uint256 public outputFeeInPoints = 0;
+
+    address public feeVaultAddress = address(0);
```

Comment:
Probably can just removed the 0 initialization here, since it will use the zero value by default